### DISCUSSION TITLE
The plan to support Kubernetes deployment

### DISCUSSION THREAD

**Discussion by niulai6** [ArroyoSystems/arroyo#53] [General]
Is there any release date plan of this feature?

**Comment by mwylde** [comment]
Yes, Kubernetes support (both as a scheduler for running pipelines and as a deployment target for the Arroyo control plane) is next on our roadmap.

**Reply by niulai6** [reply]
Thanks, will give a try when it's possible. :)

**Reply by mwylde** [reply]
Hey @xin-hao-awx -- wanted to let you know that we just shipped Kubernetes support in 0.2. The docs are here: https://doc.arroyo.dev/deployment/kubernetes. I'd love to hear what you think!

**Reply by niulai6** [reply]
This is awesome! Will give a try~
